src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -479,6 +479,11 @@ private void LogErrorOutput(string output)
         /// </remarks>
         internal abstract void LogMessages(string output, MessageImportance messageImportance);
 
+        public string GenerateResponseFileContents()
```

Comment:
@khyperia the method it's calling is `protected` so it's not directly accessible. I thought about shadowing the name but it seemed to cause more confusion that I wanted.  Open to suggestions for better names.